backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveDiskCommand.java

```diff
@@ -170,11 +170,12 @@ protected Map<String, Pair<String, String>> getExclusiveLocks() {
 
     private String getDiskIsBeingMigratedMessage() {
         if (cachedDiskIsBeingMigratedMessage == null) {
+            DiskImage diskImage = diskImageDao.get(getParameters().getImageId());
+            String diskName = getParameters().getNewAlias() != null ? getParameters().getNewAlias() :diskImage.getDiskAlias();
```

Comment:
@ArtiomDivak two comments I added before still hold:
1. diskImage might be null here, we should not get to NPE in that case
2. we retrieve data from the database even when it's not used - this should be avoided